treeherder/perf/auto_perf_sheriffing/sherlock.py

```diff
@@ -300,7 +300,7 @@ def telemetry_alert(self):
             logger.info(f"Running detection for {probe.name}")
             cdf_ts_detector = ts_detectors[probe.get_change_detection_technique()]
 
-            for platform in ("Windows", "Darwin", "Linux"):
+            for platform in ("Windows",):
```

Comment:
Is the current plan to add additional platforms, which is why we are keeping the for loop?

Reply:
Yes, we plan on adding them back but it's unclear when. We need to get it working better from the mozdetect side first.